hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowImpl.java

```diff
@@ -164,19 +165,19 @@ private TransactionMetadata preHandle(
 
             // 3. Prepare and verify signature-data
             final var signatureMap = onsetResult.signatureMap();
-            final var txBodyBytes = onsetResult.bodyBytes();
+            final var txBodyBytes = onsetResult.transaction().bodyBytes();
             final var payerSignature = verifyPayerSignature(state, context, txBodyBytes, signatureMap);
-            final var otherSignatures = verifyOtherSignatures(state, context, txBodyBytes, signatureMap);
+            final var otherSignatures =
+                    verifyOtherSignatures(state, context, PbjConverter.asBytes(txBodyBytes), signatureMap);
 
             // 4. Eventually prepare and verify signatures of inner transaction
             final var innerContext = context.getInnerContext();
             TransactionMetadata innerMetadata = null;
             if (innerContext != null) {
-                final var innerTxBodyBytes = innerContext.getTxn().toByteArray();
+                // VERIFY: the txBytes used for inner transactions is the same as the outer transaction
                 final var innerPayerSignature =
-                        verifyPayerSignature(state, innerContext, innerTxBodyBytes, signatureMap);
-                final var innerOtherSignatures =
-                        verifyOtherSignatures(state, innerContext, innerTxBodyBytes, signatureMap);
+                        verifyPayerSignature(state, innerContext, Bytes.wrap(txBytes), signatureMap);
+                final var innerOtherSignatures = verifyOtherSignatures(state, innerContext, txBytes, signatureMap);
```

Comment:
Seems inconsistent that we are using `Bytes.wrap` for one method and not for other. We should probably change to `Bytes` on line 153

Reply:
done!